classes/dac-image-sdl.bbclass

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Base class for DAC Images with SDL support"
+
+inherit  dac-image-base
```

Comment:
It is better to inherit after dac-image-wayland. This make the file simpler, handle all wayland dependencies,  and library removing.